test/integration/targets/connection_aws_ssm/aliases

```diff
@@ -0,0 +1,2 @@
+unsupported
```

Comment:
The test can be supported if you update the `runme.sh` to run a playbook that creates an EC2 instance with the SSM Agent installed and then write out an inventory file for the rest of the test to use.

Reply:
Ok got it, will add support for that. 

Currently I added `unsupported` flag as the standerd connection test are failing for SSM Connection plugin because of unicode characters. We are in process to fix the plugin so that it will support all the unicode characters.

Reply:
Hi @mattclay, 
We updated `runme.sh` which will create Linux and Windows EC2 Instaces with SSM Agent Installed, and create inventory file as well for integration test cases execution.

Can you please review the code?

Reply:
Have the unicode issues been fixed? If so, remove the `unsupported` alias so the tests will run in CI.

If not, is there an issue or PR for fixing the issues that you can link to?

Reply:
You'll also need to add `shippable/aws/group1` to the aliases when removing `unsupported`.

Reply:
Sure. @mattclay . We already did this change in the PR https://github.com/ansible/ansible/pull/49652 . So closing this PR, as all the integration tests releated changes are cherry-picked.